package.json

```diff
@@ -3,6 +3,8 @@
   "version": "0.0.1",
   "private": true,
   "devDependencies": {
+    "bourbon": "^4.2.7",
```

Comment:
yeah, I was thinking they belonged in dev dependencies since we're doing the build manually. but I can move them if you'd rather them in dependencies.